### PR TITLE
sign-plugin-docs: information of usage of a new token has been added to the docs

### LINKED ISSUE #328: Update documentation for signing a plugin (https://grafana.com/docs/grafana/latest/developers/plugins/publish-a-plugin/sign-a-plugin/#sign-a-public-plugin)

### PR DESCRIPTION
**Pull request by Ukochka**
The signing package should first try to read a new env variable called GRAFANA_ACCESS_POLICY_TOKEN and use it for signing purposes

We need to change documentation to show how to get this new GRAFANA_ACCESS_POLICY_TOKEN and how to use it

Fixes grafana/plugin-tools#328

**Special notes for your reviewer:**

I have uploaded screenshots to https://admin.grafana.com/upload/static/img/docs/plugins/

### CODE REVIEW

**Comment by Eve832** [comment]
Hey @Ukochka can you please sign the CLA? We can't merge without it.

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md
Comment (Eve832) [inline docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md:37]:
```suggestion
1. Login into your account and navigate to **My Account > Security > Access Policies**. Click **Create access policy**.
```

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md
Comment (Eve832) [inline docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md:39]:
```suggestion
   Realm: has to be your-org-name (all-stacks)
   
```

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md
Comment (Eve832) [inline docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md:44]:
```suggestion
1. Click **Create token** to create a new token.
```

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md
Comment (Eve832) [inline docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md:50]:
```suggestion

   The expiration date field is optional, though you should change tokens periodically for increased security.

   {{< figure src="/static/img/docs/plugins/create-access-policy-token.png" class="docs-image--no-shadow" max-width="650px" >}}

1. Click **Create** and save a copy of the token somewhere secure for future reference. 
```

**Review by Eve832 — CHANGES_REQUESTED** [review]

**Comment by josmperez** [comment]
I concur with Eve's suggestions. Otherwise, looks good.

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md
Comment (Ukochka) [reply docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md]: Done! Thank you!

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md
Comment (Ukochka) [reply docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md]: Done! Thank you!

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md
Comment (Ukochka) [reply docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md]: Done! Thank you!

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md
Comment (Ukochka) [reply docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md]: Done! Thank you!

File: docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md
Comment (academo) [inline docs/sources/developers/plugins/publish-a-plugin/sign-a-plugin.md:33]: `generate access token` -> `generate an access token`

**Comment by tolzhabayev** [comment]
@Ukochka due to how our documentation works ([gh actions](https://github.com/grafana/grafana/blob/main/.github/workflows/publish-technical-documentation-release.yml)), you have to backport this PR to 10.0.x by adding a `backport 10.0.x` label to it. After the merge grafanabot will create a backport PR and link it here. You can approve and merge the backport PR yourself (no need for additional approvals). 

Once merged into main the doc changes will be available at: 
- https://grafana.com/docs/grafana/next/developers/plugins/publish-a-plugin/sign-a-plugin/

Once backport is merged at:
- https://grafana.com/docs/grafana/latest/developers/plugins/publish-a-plugin/sign-a-plugin/

Both might take a while to sync and publish.

**Review by Eve832 — APPROVED** [review]

**Pull request merged**
